rosco-web/pkg_scripts/postInstall.sh

```diff
@@ -12,17 +12,20 @@ if [ -z `getent passwd spinnaker` ]; then
 fi
 
 # install packer
-PACKER_VERSION="0.12.1"
-packer_version=$(packer --version)
-packer_status=$?
-if [ $packer_status -ne 0 ] || [ "$packer_version" -ne "$PACKER_VERSION" ]; then
-  pushd .
-  cd /usr/bin
-  wget https://releases.hashicorp.com/packer/${PACKER_VERSION}/packer_${PACKER_VERSION}_linux_amd64.zip
-  apt-get install unzip -y
-  unzip -o "packer_${PACKER_VERSION}_linux_amd64.zip"
-  rm "packer_${PACKER_VERSION}_linux_amd64.zip"
-  popd
-fi
+function install_packer() {
+  PACKER_VERSION="0.12.1"
+  local packer_version=$(packer --version)
+  local packer_status=$?
+  if [[ $packer_status -ne 0 ]] || [[ "$packer_version" != "$PACKER_VERSION" ]]; then
+    pushd .
+    cd /usr/bin
+    wget https://releases.hashicorp.com/packer/${PACKER_VERSION}/packer_${PACKER_VERSION}_linux_amd64.zip
```

Comment:
Would it make sense to download this file into a directory other than `/usr/bin`? Even though we're deleting it soon after, feels a little off.